tests/simulation/test_lammps.py

```diff
@@ -0,0 +1,11 @@
+from mofa.simulation.lammps import LAMMPSRunner
+
+
+def test_LMPRunner():
+    LMPrunner = LAMMPSRunner(lammps_command="npt_tri",
```

Comment:
Code style comment: which I'm surprised flake didn't catch. Capital letters in Python are usually only used for class names.

Could you make the variables lower-case? It makes it easier for other developers (e.g., me) to quickly understand what's a ClassName and what's an instance of that class.